scripts/generate_json_terms.py

```diff
@@ -0,0 +1,361 @@
+    ],
+    # Markt, braderie of kermis
+    '0.37.0.0.0': [
+        '1.17.0.0.0', # Antiek en brocante
+        '1.66.0.0.0', # Economie
+        '1.62.0.0.0', # Gezondheid en wellness
+        '1.63.0.0.0', # Landbouw en platteland
+        '1.10.0.0.0', # Literatuur
+        '1.0.9.0.0',  # Meerdere kunstvormen
+        '1.64.0.0.0', # Milieu en natuur
+        '1.37.2.0.0', # Samenleving
+    ],
+    # Opendeurdag
+    '0.12.0.0.0': [
+        '1.2.1.0.0',  # Architectuur
+        '1.21.0.0.0', # Computer en techniek
+        '1.40.0.0.0', # Erfgoed
+        '1.41.0.0.0', # Kunst en kunsteducatie
+        '1.63.0.0.0', # Landbouw en platteland
+        '1.10.0.0.0', # Literatuur
+        '1.0.9.0.0',  # Meerdere kunstvormen
+        '1.37.2.0.0', # Samenleving
+        '1.25.0.0.0', # Wetenschap
+    ],
+    # Party of fuif
+    # '0.49.0.0.0': [],
+    # Route
+    # '0.17.0.0.0': [],
+    # Spel of quiz
+    # '0.50.21.0.0': [],
+    # Sport en beweging
+    '0.59.0.0.0': [
+        '1.51.14.0.0', # Atletiek, wandelen en fietsen
+        '1.51.13.0.0', # Bal en racketsport
+        '1.51.6.0.0',  # Fitness, gymnastiek, dans en vechtsport
+        '1.58.8.0.0',  # Lucht en motorsport
+        '1.51.12.0.0', # Omnisport en andere
+        '1.51.11.0.0', # Outdoor en Adventure sport
+        '1.51.10.0.0', # Volkssporten
+        '1.51.3.0.0',  # Zwemmen en watersport
+    ],
+    # Sportwedstrijd bekijken
+    '0.19.0.0.0': [
+        '1.51.14.0.0', # Atletiek, wandelen en fietsen
+        '1.51.13.0.0', # Bal en racketsport
+        '1.51.6.0.0',  # Fitness, gymnastiek, dans en vechtsport
+        '1.58.8.0.0',  # Lucht en motorsport
+        '1.51.12.0.0', # Omnisport en andere
+        '1.51.11.0.0', # Outdoor en Adventure sport
+        '1.51.10.0.0', # Volkssporten
+        '1.51.3.0.0',  # Zwemmen en watersport
+    ],
+    # Tentoonstelling
+    '0.0.0.0.0': [
+        '1.2.1.0.0',  # Architectuur
+        '1.1.0.0.0',  # Audiovisuele kunst
+        '1.0.2.0.0',  # Beeldhouwkunst
+        '1.0.5.0.0',  # Decoratieve kunst
+        '1.2.2.0.0',  # Design
+        '1.40.0.0.0', # Erfgoed
+        '1.0.6.0.0',  # Fotografie
+        '1.11.0.0.0', # Geschiedenis
+        '1.0.4.0.0',  # Grafiek
+        '1.0.3.0.0',  # Installatiekunst
+        '1.10.0.0.0', # Literatuur
+        '1.0.9.0.0',  # Meerdere kunstvormen
+        '1.64.0.0.0', # Milieu en natuur
+        '1.49.0.0.0', # Mode
+        '1.37.2.0.0', # Samenleving
+        '1.0.1.0.0',  # Schilderkunst
+        '1.25.0.0.0', # Wetenschap
+    ],
+    # Theatervoorstelling
+    '0.55.0.0.0': [
+        '0.52.0.0.0', # Circus
+        '1.3.5.0.0',  # Figuren en poppentheater
+        '1.3.10.0.0', # Humor en comedy
+        '1.3.4.0.0',  # Mime en bewegingstheater
+        '1.4.0.0.0',  # Musical
+        '1.5.0.0.0',  # Opera en operette
+        '1.3.1.0.0',  # Tekst- en muziektheater
+    ],
+}
+
+tree = ET.parse('../xml/term.xml')
```

Comment:
I guess there is no way to do this without starting from the XML?

I was thinking maybe we could move the actual data into this file and then use that to generate both the JSON and the XML. So that the JSON is not dependent from a legacy format that might be sunset in the future. But then we'd also have to include domains like `flandersregion` here that are not needed in the JSON, just to generate the XML 🤔 

Don't get me wrong I'm open to leaving it like this since it's probably the easiest approach. Only that the legacy format would be the single source of truth would feel a bit weird to me, and if we ever phase out the XML we'd need to change how the JSON gets created.

What do you think @willaerk @JonasVHG ?

Reply:
I think it somewhat depends on how long the two have to coexist. If this will be a long time, I would leave it like this, as we will probably be updating some of the content too, and then the single source of truth argument would be more valid, even if the source is the legacy format.
Once we phase out the XML, we can probably just keep the JSON, without the scripts.
Is that a correct assumption?

Reply:
This is only a first run. In the future, it would be best to do a refactor that generates both XML & JSON from a true source, but as of now we can only do it with the XML.

Reply:
> Once we phase out the XML, we can probably just keep the JSON, without the scripts.
> Is that a correct assumption?

Hmm that would only be easy to do (maintain it manually without scripts) if we don't embed the full terms in `otherSuggestedTerms`. Because right now that means we have "duplicate" terms in the document (once on level 1 and sometimes on level 2 in "otherSuggestedTerms")

Reply:
> This is only a first run. In the future, it would be best to do a refactor that generates both XML & JSON from a true source, but as of now we can only do it with the XML.

Okay let's leave it like it is then, no need to redo work now if we're going to change the approach anyway in the future 👍